src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -253,7 +260,60 @@
             })
             .catch(ibexa.helpers.notification.showErrorNotification);
     };
+    const validateInput = (input) => {
+        const isInputValid = !!input.value;
+        const field = input.closest('.form-group');
+        const labelNode = field.querySelector('.ibexa-label');
+        const errorNode = field.querySelector('.ibexa-form-error');
+
+        input.classList.toggle('is-invalid', !isInputValid);
+
+        if (errorNode) {
+            const fieldName = labelNode.innerHTML;
+            const errorMessage = ibexa.errors.emptyField.replace('{fieldName}', fieldName);
+
+            errorNode.innerHTML = !isInputValid ? errorMessage : '';
+        }
+
+        if (isEditFormValid && !isInputValid) {
+            isEditFormValid = false;
+        }
+    };
+    const validateForm = () => {
+        const inputsToValidate = editForm.querySelectorAll('.ibexa-input[required]:not([disabled]):not([hidden])');
```

Comment:
what would you say on adding this selecto to some const? you use it few times here

Reply:
I'm not a big fan of this type of variables but ok :)